wolfcrypt/src/sha.c

```diff
@@ -812,6 +864,9 @@ int wc_ShaFinal(wc_Sha* sha, byte* hash)
     else {
         ret = esp_sha_digest_process(sha, 1);
     }
+#elif defined(WOLFSSL_USE_ESP32C3_CRYPT_HASH_HW)
```

Comment:
This and the `#else case` are the same.  Please combine and just document with a comment.

Reply:
Yes, they were the same on purpose as a  placeholder for the `-C3`, but I agree that's probably not a great idea. Combined and comment added as requested.